commit dab5538f0bfc4cca76396f5d510e2df6f5350d4c
Author: David Rowley <drowley@postgresql.org>
Date:   Thu Oct 12 21:15:28 2023 +1300

    Doc: fix grammatical errors for enable_partitionwise_aggregate
    
    Author: Andrew Atkinson
    Reviewed-by: Ashutosh Bapat
    Discussion: https://postgr.es/m/CAG6XLEnC%3DEgq0YHRic2kWWDs4xwQnQ_kBA6qhhzAq1-pO_9Tfw%40mail.gmail.com
    Backpatch-through: 11, where enable_partitionwise_aggregate was added

diff --git a/doc/src/sgml/config.sgml b/doc/src/sgml/config.sgml
index 924309af26d..3839c72c868 100644
--- a/doc/src/sgml/config.sgml
+++ b/doc/src/sgml/config.sgml
@@ -5269,17 +5269,18 @@ ANY <replaceable class="parameter">num_sync</replaceable> ( <replaceable class="
       </indexterm>
       </term>
       <listitem>
        <para>
         Enables or disables the query planner's use of partitionwise grouping
-        or aggregation, which allows grouping or aggregation on a partitioned
-        tables performed separately for each partition.  If the <literal>GROUP
-        BY</literal> clause does not include the partition keys, only partial
-        aggregation can be performed on a per-partition basis, and
-        finalization must be performed later.  Because partitionwise grouping
-        or aggregation can use significantly more CPU time and memory during
-        planning, the default is <literal>off</literal>.
+        or aggregation, which allows grouping or aggregation on partitioned
+        tables to be performed separately for each partition.  If the
+        <literal>GROUP BY</literal> clause does not include the partition
+        keys, only partial aggregation can be performed on a per-partition
+        basis, and finalization must be performed later.  Because
+        partitionwise grouping or aggregation can use significantly more CPU
+        time and memory during planning, the default is
+        <literal>off</literal>.
        </para>
       </listitem>
      </varlistentry>
 
      <varlistentry id="guc-enable-presorted-aggregate" xreflabel="enable_presorted_aggregate">
